final/app.js

```diff
@@ -0,0 +1,122 @@
+/*
+GAME RULES:
+
+- The game has 2 players, playing in rounds
+- In each turn, a player rolls a dice as many times as he whishes. Each result get added to his ROUND score
+- BUT, if the player rolls a 1, all his ROUND score gets lost. After that, it's the next player's turn
+- The player can choose to 'Hold', which means that his ROUND score gets added to his GLBAL score. After that, it's the next player's turn
+- The first player to reach 100 points on GLOBAL score wins the game
+
+*/
+
+var scores, roundScore, activePlayer, gamePlaying;
+
```

Comment:
General comments:
1) You should update the Readme.md to include the purpose of what you're doing, how somebody would clone your repo and how somebody would run the code.
2) There is no .gitignore file, this means that files like .zip files are being committed. This is really bad practice and could end up with secrets being committed by accident. The .gitignore file is possibly the most important file of any project.
3) Code is written using JS5 rather than JS2015 onwards. Once you understand the differences in JS2015 it would be worthwhile re-writing this code to make use of the best practice.
4) Code includes the original code and your code - this is confusing, the whole point of a version control system (like git) is that it manages the changes for you.